plugins/logs/plugin_test.go

```diff
@@ -1742,8 +1742,8 @@ func compareLogEvent(t *testing.T, actual []byte, exp EventV1) {
 
 func testStatus() *bundle.Status {
 
-	tDownload, _ := time.Parse("2018-01-01T00:00:00.0000000Z", time.RFC3339Nano)
-	tActivate, _ := time.Parse("2018-01-01T00:00:01.0000000Z", time.RFC3339Nano)
+	tDownload, _ := time.Parse(time.RFC3339Nano, "2018-01-01T00:00:00.0000000Z")
+	tActivate, _ := time.Parse(time.RFC3339Nano, "2018-01-01T00:00:01.0000000Z")
```

Comment:
Just a note, it's curious that this hasn't caused issues. So, I'd guess the tests don't care about the actual timestamps. 💭

Reply:
I'm pretty sure you're right. I suspect that even though as-it-was scheduled the events for the same time, that since they were ordered in the way we expect it didn't matter.